aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -505,6 +505,30 @@ void minimum_kernel(TensorIterator& iter) {
   }
 }
 
+void fmax_kernel(TensorIterator& iter) {
+  if (isFloatingType(iter.common_dtype())) {
+    AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.dtype(), "fmax_cpu", [&]() {
+      cpu_kernel(iter,
+        [](scalar_t a, scalar_t b) -> scalar_t {return std::fmax(a, b);
```

Comment:
I believe the linter will complain that the return statement is on the same line. Could you move it to its own line.

Reply:
Addressed!